File: EducationApp.DataAccessLayer/Models/Filter/PrintingEdition/PrintingEditionFilterModel.cs
using EducationApp.DataAccessLayer.Helpers.Base;
using System.Collections.Generic;
using static EducationApp.DataAccessLayer.Entities.Enums.Enums;

namespace EducationApp.DataAccessLayer.Helpers.PrintingEditionFilter
{
    public class PrintingEditionFilterModel : BaseFilterModel
    {
        public PrintingEditionSortType PrintingEditionSortType { get; set; }
        public List<TypeProduct> TypeProduct { get; set; }
        public CurrencyType CurrencyType { get; set; }
        public decimal MinPrice { get; set; }
        public decimal MaxPrice { get; set; }
    }
}

File: ClientApp/src/app/shared/models/printing-editions/PrintingEditionFilterModel.ts
import { BaseFilterModel } from "src/app/shared/models/Base/BaseFilterModel";
import { PrintingEditionSortType } from "src/app/shared/enums/PrintingEditionSortType";
import { ProductType } from 'src/app/shared/enums/ProductType';
import { CurrencyType } from 'src/app/shared/enums/CurrencyType';

export class PrintingEditionFilterModel extends BaseFilterModel {

    printingEditionSortType: PrintingEditionSortType;
    typeProduct: Array<ProductType>;
    currencyType: CurrencyType;
    minPrice: number;
    maxPrice: number;

}